library/core/src/slice/iter.rs

```diff
@@ -270,6 +270,13 @@ pub(super) trait SplitIter: DoubleEndedIterator {
 ///
 /// This struct is created by the [`split`] method on [slices].
 ///
+/// # Example
```

Comment:
This should be `Examples` for consistency, not `Example`.

Reply:
Ah true. I might have make the same mistake in some other places...